site/docs/components/card/examples.mdx

```diff
@@ -72,6 +72,34 @@ When using images in cards, set the image to span the card's entire width. You c
 - By default, the card components accent color uses `--salt-container-primary-borderColor` or `--salt-container-secondary-borderColor` depending on the variant.
 - When there is a need to associate a brand or product color to the accent bar, you can override `--saltCard-accent-color` or `--saltLinkCard-accent-color` with the desired color.
 
+</LivePreview>
+<LivePreview componentName="card" exampleName="InteractableCardExample" displayName="Interactable Card">
+## Interactable card
+
+- Use an Interactable card when the entire card functions as a button, or when the entire card needs to be selectable from a group of cards.
+
+</LivePreview>
+
+<LivePreview componentName="card" exampleName="DisabledInteractableCard" >
+## Disabled interactable card
+
+- The Interactable card component can be disabled to prevent the user from interacting with it, i.e. if the option is temporarily unavailable
```

Comment:
```suggestion
- The Interactable card component can be disabled to prevent the user from interacting with it, e.g., if the option is temporarily unavailable
```